src/linkedList.js

```diff
@@ -0,0 +1,188 @@
+class Node {
+
+  constructor( data ) {
+    this._data = data
+    this._next = null
+  }
+
+  data() {
+    return this._data
+  }
+
+  next() {
+    return this._next
+  }
+
+}
+
+export default class LinkedList {
+
+  constructor() {
+    this._head = null
+    this._tail = null
+    this._size = 0
+  }
+
+  getHeadNode() {
+    return this._head
+  }
+
+  getTailNode() {
+    return this._tail
+  }
+
+  contains( data ) {
+    return this.find( data ) !== -1
+  }
+
+  find( data ) {
+    if ( !this.isEmpty() ) {
+      let node = this.getHeadNode()
+      let nextNode
+      do {
+        nextNode = node
+        if ( node.data() === data ) {
+          return node
+        }
+        else {
+          node = node.next()
+        }
+      } while ( nextNode.next() !== null )
+
+    }
+    return -1
+  }
+
+// Helper function
```

Comment:
eh, your code is very clear and concise already...I don't particularly see the need for comments. I personally don't like comments in my final code...but that's just me :)